### PR TITLE
Add PauseOnHover for autoplay feature

### LINKED ISSUE #4: feat: add option pauseOnHover
- [x] Add option `pauseOnHover`
- [x] True by default
- [x] Add a note to README

### PR DESCRIPTION
**Pull request by vuquangpham**
### Pause On Hover

I have added a little bit of code for the autoplay feature.

With that code, we can control the autoplay feature by hovering over it. You can turn on this option by passing an option `pauseOnHover` through HTML attribute or Javascript object. And when the option `pauseOnHover` is false, I also removed the event listener for optimizing performance.

### CODE REVIEW

File: src/_index.js [hunk 1]
```diff
@@ -94,6 +103,12 @@ class Slider{
         this.isPlay = true;
         checkAutoplay(this);
 
+        // pauseOnHover (priority: attribute > options)
+        if(this.wrapper.hasAttribute(this._attr.pauseOnHover)){
+            const isPauseOnHover = this.wrapper.getAttribute(this._attr.pauseOnHover).toLowerCase().trim();
+            this.options.pauseOnHover = isPauseOnHover === "true";
```
Comment (phucbm) [inline src/_index.js:109]: Whenever the attribute for `pauseOnHover` appear in the wrapper element, we will have `pauseOnHover: true`. 

Don't care what the value is. 

This is just the same with the `swipe` option which you might have had a look already, haven't you?

File: web/html/home.html [hunk 2]
```diff
@@ -84,6 +84,8 @@ <h2>#3 Autoplay</h2>
         <button data-select="play" data-id="slider-3" class="btn green disabled">Play</button>
         <button data-select="prev" data-id="slider-3" class="btn">Previous</button>
         <button data-select="next" data-id="slider-3" class="btn">Next</button>
+        <button data-select="remove-poh" data-id="slider-3" class="btn">Remove PauseOnHover</button>
+        <button data-select="register-poh" data-id="slider-3" class="btn">Add PauseOnHover</button>
```
Comment (phucbm) [inline web/html/home.html:88]: There should be only one clickable button at a time. You can either use `.disabled` class (see Pause and Play button) or display none/block as we also don't have so much spacing here to have so many buttons.

**Review by phucbm — CHANGES_REQUESTED** [review]
Please check my comments.

**Comment by vuquangpham** [comment]
I have fixed it. Please check again!

**Review by phucbm — APPROVED** [review]

**Pull request merged**